controllers/drplacementcontrol_controller.go

```diff
@@ -401,11 +401,7 @@ func DRPCsUsingDRCluster(k8sclient client.Client, log logr.Logger, drcluster *rm
 	for i := range drpolicies.Items {
 		drpolicy := &drpolicies.Items[i]
 
-		for _, clusterName := range drpolicy.Spec.DRClusters {
-			if clusterName != drcluster.GetName() {
-				continue
-			}
-
+		if rmnutil.DrpolicyContainsDrcluster(drpolicy, drcluster.GetName()) {
```

Comment:
Maybe rmnutil.DrpolicyHasDrcluster? Little bit shorter and looks nice to me.

Reply:
@nirs the name of this helper was Shyam's suggestion [here](https://github.com/RamenDR/ramen/pull/1218#discussion_r1511085084)

Reply:
Name suggestions can always be changed, no issues. We may want to think of implementing methods to DRPolicy and other resources, such that we could actually invoke something like `drpolicy.ContainsCluster(name)` instead.

Reply:
Having methods on drpolicy would be nicer, but lets start with this change.